lightning_pose/data/datasets.py

```diff
@@ -298,13 +270,16 @@ def __init__(
         self.compute_heatmaps()
 
     @property
-    def output_shape(self):
+    def output_shape(self) -> tuple:
         return (
             self.height // 2**self.downsample_factor,
             self.width // 2**self.downsample_factor,
         )
 
-    def compute_heatmap(self, example_dict: BaseExampleDict):
+    def compute_heatmap(
+        self,
+        example_dict: BaseLabeledExampleDict
+    ) -> TensorType["num_keypoints", "heatmap_height", "heatmap_width"]:
         """Compute 2D heatmaps from arbitrary (x, y) coordinates."""
 
         if self.do_context:
```

Comment:
I think we can eliminate this and use `self.height` and `self.width` properties inherited from `BaseTrackingDataset`, assuming all images in the dataset are resized to be of the same size.

Reply:
yes this is a good idea